Libs/MRML/Widgets/qMRMLChartWidget.cxx

```diff
@@ -161,3 +172,14 @@ vtkMRMLColorLogic* qMRMLChartWidget::colorLogic()const
   Q_D(const qMRMLChartWidget);
   return d->ChartView->colorLogic();
 }
+
+#ifndef MRML_WIDGETS_HAVE_WEBKIT_SUPPORT
+//---------------------------------------------------------------------------
+void qMRMLChartWidget::onAppAboutToQuit()
```

Comment:
Would be best to move this slot into the pimpl

Reply:
You cannot have a slot in a pimpl when the class declaration is in the cpp. I would have to move the class declaration in a private header so Qt can run moc on it.
I thought against since it would involve moving a lot of code, but it might be worth it ? What do you think ?